src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -55,12 +68,48 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
             body = needsParentheses(actualExpression) ? createParen(actualExpression) : actualExpression;
             suppressLeadingAndTrailingTrivia(body);
             copyLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            copyTrailingAsLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            // Copy the trailing comments after the return statement
+            // Add them to our trailingCommentsHolder to use later
+            copyTrailingComments(returnStatement, trailingCommentsHolder, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+
+
+            // If there are trailing comments
+            if (trailingCommentsHolder && trailingCommentsHolder.emitNode && trailingCommentsHolder.emitNode.trailingComments) {
+
+                // First we format our comment
+                // ex. " trailing comment " -> "/* trailing comment */"
+                const formattedComment = formatSynthesizedComment(trailingCommentsHolder.emitNode.trailingComments[0]);

```

Comment:
I don't like that we're grabbing just the first comment based on the index `0`. This could probably be done in a better way. Plus, what happens if we have multiple comments? I think this would break.